rust/crd/src/lib.rs

```diff
@@ -957,6 +974,8 @@ pub struct DriverConfig {
     #[fragment_attrs(serde(default))]
     #[fragment_attrs(schemars(schema_with = "pod_overrides_schema"))]
     pub pod_overrides: PodTemplateSpec,
+    #[fragment_attrs(serde(default))]
+    pub jvm_security: HashMap<String, Option<String>>,
```

Comment:
I think the current CRD change is not ideal due to the following reasons:

1. It differs from other other operators. Maybe we want to put it below spec.driver.configOverrides."security.properties"
2. The way the defaults currently work means that all defaults are lost once the users adds anything to the Hashmap. IMHO we should use the merge mechanism, so that user input is merged on top of our defaults instead of replacing them

Reply:
> 2. The way the defaults currently work means that all defaults are lost ...

Actually they are merged (and retained) because of this:
https://github.com/stackabletech/spark-k8s-operator/blob/22545ef71fd62e635a66931079ff832dde07ae5e/rust/crd/src/lib.rs#L666

Reply:
Sorry, you're right, I'm stupid. Please dismiss point 2.)